CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+#mainframer Change Log
+
+## Version 1.0.1
```

Comment:
I would prefer [Square format](https://github.com/square/retrofit/blob/master/CHANGELOG.md) with dates in subsections.